src/main/java/com/google/javascript/gents/Options.java

```diff
@@ -75,7 +89,17 @@ private void setIdeMode(final CompilerOptions options) {
     options.setIdeMode(true);
   }
 
-  Options(String[] args) throws CmdLineException {
+  private Map<String, String> getExternsMap() throws FileNotFoundException {
+    if (this.externsMapFile != null) {
+      Type mapType = new TypeToken<Map<String, String>>(){}.getType();
+      JsonReader reader = new JsonReader(new FileReader(externsMapFile));
+      return new Gson().fromJson(reader, mapType);
+    } else {
+      return ImmutableMap.of();
+    }
+  }
+
+  Options(String[] args) throws CmdLineException, FileNotFoundException {
```

Comment:
Nit/optional: maybe it'd be nicer for code calling into this to convert the `FileNotFoundException` into a `CmdLineException`, so it doesn't have to special case the catch?